frontend/talentsearch/src/js/components/search/EstimatedCandidates.tsx

```diff
@@ -75,6 +75,23 @@ const EstimatedCandidates: React.FunctionComponent<
                 </>
               )}
             </p>
+            {candidateCount > 0 && (
+              <a
+                href="#results"
+                title="View the pools that contain matching talent."
```

Comment:
This needs to be wrapped in `intl.formatMessage`

Reply:
I think it may be better to remove the `title` attribute entirely since it has some accessibility concerns.

REF: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns

Reply:
Doh!  Localizing was most of this PR.  :facepalm: Good catch.  Added in [Localize link title](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4196/commits/fd183c4748661b4ca6bad9d2c83c2564020f625a).

Reply:
Ah, right, the link text is descriptive enough anyway.  Removed in [Remove title](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4196/commits/37d7d372b9e2244ec6e627b7b3e54d80c421dd0b).